examples/react/basic/src/gen/eliza-BigIntService_connectquery.ts

```diff
@@ -41,9 +41,12 @@ export const BigIntService = {
   },
 } as const;
 
+const queryService = createQueryService({ service: BigIntService });
+
 /**
  * @generated from rpc connectrpc.eliza.v1.BigIntService.Count
  */
-export const count = createQueryService({
-  service: BigIntService,
-}).count;
+export const count = {
+  ...queryService.count,
+  ...createHooks(queryService.count),
+};
```

Comment:
So these changes to protoc-gen-connect-query are not a breaking change, but we're adding new properties, if I understand correctly.

Do we explain anywhere what we add?